Gemfile.lock

```diff
@@ -266,125 +268,127 @@ GEM
       minitest (>= 5.1)
       tzinfo (~> 2.0)
       zeitwerk (~> 2.3)
-    addressable (2.8.4)
+    addressable (2.8.6)
       public_suffix (>= 2.0.2, < 6.0)
-    airbrake (11.0.1)
+    airbrake (11.0.3)
       airbrake-ruby (~> 5.1)
-    airbrake-ruby (5.2.0)
+    airbrake-ruby (5.2.1)
       rbtree3 (~> 0.5)
     ansible (0.2.2)
     ar_multi_threaded_transactional_tests (0.5.0)
       activerecord (>= 4.2.0, < 6.2.0)
-    ast (2.4.1)
+    ast (2.4.2)
     attr_encrypted (3.1.0)
       encryptor (~> 3.0.0)
-    audited (5.4.2)
+    audited (5.4.3)
       activerecord (>= 5.0, < 7.2)
       request_store (~> 1.2)
-    autoprefixer-rails (9.4.8)
+    autoprefixer-rails (9.8.6.5)
       execjs
-    awesome_print (1.6.1)
-    aws-eventstream (1.2.0)
-    aws-partitions (1.329.0)
-    aws-sdk-core (3.100.0)
-      aws-eventstream (~> 1, >= 1.0.2)
-      aws-partitions (~> 1, >= 1.239.0)
+    awesome_print (1.9.2)
+    aws-eventstream (1.3.0)
+    aws-partitions (1.887.0)
+    aws-sdk-core (3.191.0)
+      aws-eventstream (~> 1, >= 1.3.0)
+      aws-partitions (~> 1, >= 1.651.0)
+      aws-sigv4 (~> 1.8)
+      jmespath (~> 1, >= 1.6.1)
+    aws-sdk-ecr (1.69.0)
+      aws-sdk-core (~> 3, >= 3.191.0)
       aws-sigv4 (~> 1.1)
-      jmespath (~> 1.0)
-    aws-sdk-ecr (1.3.0)
-      aws-sdk-core (~> 3)
-      aws-sigv4 (~> 1.0)
-    aws-sdk-kms (1.23.0)
-      aws-sdk-core (~> 3, >= 3.58.0)
+    aws-sdk-kms (1.77.0)
+      aws-sdk-core (~> 3, >= 3.191.0)
       aws-sigv4 (~> 1.1)
-    aws-sdk-s3 (1.45.0)
-      aws-sdk-core (~> 3, >= 3.58.0)
+    aws-sdk-s3 (1.143.0)
+      aws-sdk-core (~> 3, >= 3.191.0)
       aws-sdk-kms (~> 1)
-      aws-sigv4 (~> 1.1)
-    aws-sigv4 (1.6.1)
+      aws-sigv4 (~> 1.8)
+    aws-sigv4 (1.8.0)
       aws-eventstream (~> 1, >= 1.0.2)
-    base64 (0.1.1)
+    base64 (0.2.0)
+    bigdecimal (3.1.6)
     binding_of_caller (0.8.0)
       debug_inspector (>= 0.0.1)
-    bootsnap (1.4.5)
-      msgpack (~> 1.0)
+    bootsnap (1.18.3)
+      msgpack (~> 1.2)
     bootstrap-sass (3.4.1)
       autoprefixer-rails (>= 5.2.1)
       sassc (>= 2.0.0)
     bootstrap3-datetimepicker-rails (4.17.47)
       momentjs-rails (>= 2.8.1)
     brakeman (5.4.1)
     builder (3.2.4)
-    bundler-audit (0.9.0.1)
+    bundler-audit (0.9.1)
       bundler (>= 1.2.0, < 3)
       thor (~> 1.0)
     byebug (8.2.5)
-    chef-utils (18.1.29)
+    chef-utils (18.4.2)
       concurrent-ruby
-    coderay (1.1.1)
+    coderay (1.1.3)
     commonmarker (0.23.10)
-    concurrent-ruby (1.2.2)
-    connection_pool (2.2.1)
-    crack (0.4.3)
-      safe_yaml (~> 1.0.0)
+    concurrent-ruby (1.2.3)
+    connection_pool (2.4.1)
+    crack (0.4.6)
```

Comment:
### [crack](https://github.com/jnunemaker/crack)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 0.4.3 → 0.4.6

[_[change-log](https://github.com/jnunemaker/crack/blob/master/History), [source-code](https://github.com/jnunemaker/crack), [gem-diff](https://my.diffend.io/gems/crack/0.4.3/0.4.6)_]





<details>
<summary>Commits</summary>

A change of **24** commits. See the full changes on [the compare page](https://github.com/jnunemaker/crack/compare/v0.4.3...v0.4.6).

These are the first 10 commits:
- (a7189fe) [remove safe_yaml](https://github.com/jnunemaker/crack/commit/a7189fef728ce522cb03f62b18848cfc50058245)
- (261962a) [Exclude tests (and fixtures) from bundled gem](https://github.com/jnunemaker/crack/commit/261962ae56f0c4216d0be6b8c65037ec5c853a70)
- (1183581) [Merge pull request #61 from greysteil/dont-bundle-tests](https://github.com/jnunemaker/crack/commit/1183581ded7d7ee6b427778eef430d3d10000e04)
- (4ecf9cf) [Merge pull request #59 from bf4/master](https://github.com/jnunemaker/crack/commit/4ecf9cf5f721dea61b8fa86280180142ead35312)
- (579acb3) [Add REXML to gem dependency](https://github.com/jnunemaker/crack/commit/579acb39c5597b5a4353181a5cf967c148625275)
- (288371d) [Bump version to 0.4.4; update history](https://github.com/jnunemaker/crack/commit/288371d0444ba64bde1b082f340fcc67eeb9e64e)
- (244855f) [GitHub is HTTPS by default](https://github.com/jnunemaker/crack/commit/244855f6ee77f85fd78a2e303de484fd17c60edf)
- (f61a62e) [Merge pull request #64 from amatsuda/https](https://github.com/jnunemaker/crack/commit/f61a62eb5b2ad877950831c5a69c6e230c6c73f2)
- (c61172b) [Merge pull request #62 from koic/add_rexml_to_gem_dependenci…](https://github.com/jnunemaker/crack/commit/c61172bf32e1769748fded156c2f2fc03dac69c1)
- (7ec6b55) [Bump to 0.4.5](https://github.com/jnunemaker/crack/commit/7ec6b55efd0055cc04e30be0fa828e0902c45c7b)


</details>